refl1d/experiment.py

```diff
@@ -323,6 +337,17 @@ def save_refl(self, basename):
             theory = self.reflectivity(interpolation=self.interpolation)
             self.probe.save(filename=basename + "-refl-interp.dat", theory=theory)
 
+    def register_webview_plot(
+        self, plot_title: str, plot_function: WebviewPlotFunction, change_with: Literal["parameter", "uncertainty"]
+    ):
+        # Plot function syntax: f(model, problem, state)
```

Comment:
Now the plot function syntax depends on the function: for "parameter" functions it's just `f(model, problem)` and for "uncertainty" functions it's `f(model, problem, state, n_samples)`

This comment should probably either be removed or modified.

Reply:
Resolved by https://github.com/reflectometry/refl1d/pull/199/commits/0743a4a794c0fedff8509ddcb73cebeb5a0442c6